tests/lib/rules/images-no-direct-imports.js

```diff
@@ -0,0 +1,104 @@
+const {RuleTester} = require('eslint');
+const {fixtureFile} = require('../../utilities');
+const rule = require('../../../lib/rules/images-no-direct-imports');
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module',
+  },
+});
+
+function errors(type) {
+  return [
+    {
+      type,
+      message:
+        'Do not import image files directly, instead import from the index of the folder containing the image',
```

Comment:
We don't do it too often in other rules, but you can avoid the repetition here by using a messageId. https://eslint.org/docs/developer-guide/working-with-rules#messageids